internal/kafka/command_cluster_create.go

```diff
@@ -276,6 +288,10 @@ func setClusterConfigCku(cluster *cmkv2.CmkV2Cluster, cku int32) {
 	cluster.Spec.Config.CmkV2Dedicated.Cku = cku
 }
 
+func setClusterNetwork(cluster *cmkv2.CmkV2Cluster, network *cmkv2.EnvScopedObjectReference) {
+	cluster.Spec.Network = network
+}
```

Comment:
Does this need to be a helper function? It's clear what it does and is only one line of code.